feedthefox/base/templates/jinja2/base.html

```diff
@@ -159,7 +150,6 @@
 
     <script src="{{ static('js/jquery-2.1.1.min.js') }}"></script>
     <script src="{{ static('js/bootstrap.min.js') }}"></script>
-    <script src="https://login.persona.org/include.js"></script>
     <script src="{{ static('js/auth.js') }}"></script>
```

Comment:
Probably you need to remove `auth.js` both from here and the project in general.